lib/rom/factory/constants.rb

```diff
@@ -7,5 +7,11 @@ def initialize(name)
         super("Factory +#{name}+ not defined")
       end
     end
+
+    class UnknownFactoryAttributes < StandardError
```

Comment:
I felt like `ROM::Factory::UnknownFactoryAttributes` was a bit nicer for the user to see than `Rom::Factory::Builder::Persistable::UnknownFactoryAttributes`